Rust/Rust.sublime-syntax

```diff
@@ -238,9 +238,10 @@ contexts:
         1: punctuation.definition.annotation.rust
         2: punctuation.section.group.begin.rust
       push:
+        - meta_scope: meta.annotation.rust
```

Comment:
Indeed, thanks! Done.

(I'm not quite clear why `punctuation.definition.annotation.rust` from the captures still also gets the `meta.annotation.rust` scope. Do the scopes in `captures` automatically get the `meta_scope` as well?)